src/Components/Web.Extensions/src/Microsoft.AspNetCore.Components.Web.Extensions.csproj

```diff
@@ -6,11 +6,11 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <RootNamespace>Microsoft.AspNetCore.Components</RootNamespace>
     <Nullable>enable</Nullable>
+    <IsShipping>false</IsShipping>
```

Comment:
Why is this no longer shipping?  What's still in it?

Reply:
One of the outcomes of API review. We’ve moved out the features that are ready to ship into other packages. The one remaining thing here - head tag management - is staying as experimental because there are uncertain design questions and we don’t want to commit to supporting it as of 5.0.

Reply:
BTW head tag management is things like the <Title> component for setting page title.